client/src/GlobalStyle.js

```diff
@@ -14,7 +14,7 @@ const GlobalStyle = createGlobalStyle`
 }
 
 body,
-button {
+button, textarea {
```

Comment:
Do you really want to change all buttons and textareas? Why don't you build a component where you remove said properties and branch out new Buttons from there?
html (baseline button) => styled Button
html => baseline Button => styled Button

Reply:
I actually just set font and color which i do want for all buttons and textareas :)